### DISCUSSION TITLE
How to remove the ending sentence audio distortion?

### DISCUSSION THREAD

**Discussion by Risu** [coqui-ai/TTS#3407] [General Q&A]
It sounds like the audio input switching low volume distortion like ..pop...pop.. between every sentence. Hard to notice at first, but it is always the same at the end of every sentence.

**Comment by unifirer** [comment]
seems like its fixed, update

**Reply by Risu** [reply]
In 0.22.0 with disabled "split_sentences" yes, only the end of the file contain distortion, so each file have to be trimmed. I don't really consider this as solved, but more like a workaround.
